src/redux/books/booksSlice.js

```diff
@@ -1,42 +1,92 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const getBooks = createAsyncThunk('books/getBooks', async () => {
+  const response = await axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/hcbcyPwKLexrxoT378qc/books');
+  return response.data;
+});
+
+export const addBook = createAsyncThunk('books/addBook', async (book) => {
+  const result = await axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/hcbcyPwKLexrxoT378qc/books', book);
+  return result.data;
+});
+
+export const removeBook = createAsyncThunk('books/removeBook', async (id) => {
+  await axios.delete(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/hcbcyPwKLexrxoT378qc/books/${id}`);
+  return id;
+});
```

Comment:
- [ ] I have faced the same issue that the previous reviewer faced regarding removing functionality :+1: You can see this app behavior by following the steps :arrow_down:

1. Create a book (Don't refresh your page after book creation)
2. Remove the book
3. After removing the book kindly refresh your page.